lib/matplotlib/patches.py

```diff
@@ -1657,6 +1669,13 @@ def get_angle(self):
 
     angle = property(get_angle, set_angle)
 
+    def get_corners(self):
+        """
+        Return the corners of the ellipse bounding box, moving anti-clockwise
+        from the lower left.
```

Comment:
Is it really lower left? Can't `angle` be over 90?

Reply:
Yes, it's lower left in the de-rotated frame. I'm not sure what the best way to phrase this is - perhaps just "from the lower left before rotation is applied."?

Reply:
Would it better and clear enough to use "anchor point" (`xy`), as this term is used in the `Rectangle` patch docstring?

Reply:
I don't think that works for `Ellipse` though, since the "anchor point" is the center.

Reply:
Indeed, it isn't sensible for the `Ellipse` patch - sorry I read too quickly!
Is it worth having a `get_corners` property as public API for the `Ellipse` patch? Maybe there could a private API `_get_corners` for both (`Rectangle` and `Ellipse`) and `Rectangle` patch could have the public `get_corners` as a convenience property?